front/src/plugins/kakao-map.js

```diff
@@ -129,6 +130,10 @@ const KakaoMap = {
         };
       });
     };
+
+    Vue.prototype.$watchMapCenterChange = (func) => {
+      this.map.addListener(EVENT_TYPE.CENTER_CHANGE, func);
+    };
```

Comment:
저는 EVENT_TYPE의 의존성을 플러그인 내부가 아니라 외부에서 주입해줘야한다고 생각해요.
```suggestion
    Vue.prototype.$addEventToMap = (eventType, func) => {
      this.map.addListener(eventType, func);
    };
```
이런건 어때요?